code/modules/mob/living/silicon/robot/robot_hud.dm

```diff
@@ -0,0 +1,132 @@
+/mob/living/silicon/robot/handle_regular_hud_updates()
```

Comment:
couldn't we move much of this file up to a living level?, e.g.: processing huds, blindness, blurry, etc.

Reply:
Maybe, but should that really be bundled in the same PR that nuked mommi's life.dm?